app/helpers/cards_helper.rb

```diff
@@ -0,0 +1,10 @@
+module CardsHelper
+  def cart_item(card)
+    content_tag(:td) { card.original_text }
+    content_tag(:td) { card.translated_text }
+    content_tag(:td) { card.review_date.strftime('%d.%m.%Y %H:%M:%S') }
+    content_tag(:td) { link_to 'Редактировать', edit_card_path(card) }
+    content_tag(:td) { ink_to 'Удалить', card, method: :delete,
+                      data: { confirm: 'Удалить запись?' } }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 8, 60 should be on its own line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.